Tests/ZIPFoundationTests/ZIPFoundationWritingTests.swift

```diff
@@ -329,4 +329,100 @@ extension ZIPFoundationTests {
         let nonUpdatableArchive = Archive(url: nonUpdatableArchiveURL, accessMode: .update)
         XCTAssertNil(nonUpdatableArchive)
     }
+
+    func testUniqueTemporaryDirectoryURL() {
+        let archive = self.archive(for: #function, mode: .create)
+        var tempURLs = Set<URL>()
+        defer {
+            for url in tempURLs {
+                try? FileManager.default.removeItem(at: url)
+            }
+        }
+        // We choose 2000 temp directories to test workaround for http://openradar.appspot.com/50553219
+        for _ in 1...2000 {
```

Comment:
Nice find 🙂. 
I saw that you already got a reply - quite fast for a Radar in pre-WWDC season.

Reply:
Nice to have a reply for sure... I'm not in love with the response, however.